modules/security-context-constraints-psa-alert-eval.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * authentication/understanding-and-managing-pod-security-admission.adoc
+
+:_content-type: PROCEDURE
+[id="security-context-constraints-psa-alert-eval_{context}"]
+= Identifying pod security violations
+
+The `PodSecurityViolation` alert does not provide details on which workloads are causing pod security violations. You can identify the affected workloads by reviewing the Kubernetes API server audit logs. This procedure uses `must-gather` to gather the audit logs and then searches for the `pod-security.kubernetes.io/audit-violations` annotation.
+
+.Prerequisites
+
+* You have installed `jq`.
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. To gather the audit logs, enter the following command:
++
+[source,terminal]
+----
+$ oc adm must-gather -- /usr/bin/gather_audit_logs
+----
+
+. To output the affected workload details, enter the following command:
++
+[source,terminal]
+----
+$ zgrep -h pod-security.kubernetes.io/audit-violations must-gather.local.<archive_id>/quay*/audit_logs/kube-apiserver/*log.gz | jq -r 'select(.annotations["pod-security.kubernetes.io/audit-violations"] != null).objectRef | .namespace + " " + .name + " " + .resource' | sort | uniq -c
```

Comment:
can we split this bash command into multiple lines with about 80 characters per line to improve clarity in the final text?

Reply:
I think we may want to extend the query to only take the `resource=pod` into account

Reply:
So I had actually looked at that, but some of the parts are so long that there wasn't much that could be done. So I split as best I could, and this is what it looks like now:

```
$ zgrep -h pod-security.kubernetes.io/audit-violations must-gather.local.<archive_id>/quay*/audit_logs/kube-apiserver/*log.gz \
  | jq -r 'select(.annotations["pod-security.kubernetes.io/audit-violations"] != null).objectRef | .namespace + " " + .name + " " + .resource' \
  | sort | uniq -c
```
Let me know if you know of any better way that it could be split up more

Reply:
@stlaz Can you show me how the query would look if we added the resource=pod piece?

Reply:
it's the first query in https://access.redhat.com/solutions/6976583